UnicornRender/video/include/unicorn/video/Transform.hpp

```diff
@@ -109,13 +109,13 @@ class Transform
     UNICORN_EXPORT void Scale(glm::vec3 scale);
 
     /**
-     * @brief Translates by axes
+     * @brief Translates considering current orientation
      *
      * @attention Don't forget to recalculate @sa UpdateTransformMatrix to commit all changes to the object
      *
      * @param[in] distance per component distance
      */
-    UNICORN_EXPORT void TranslateLocal(glm::vec3 distance);
+    UNICORN_EXPORT void TranslateByBasis(glm::vec3 distance);
```

Comment:
I don't really get what this function does and how does it differ from TranslateWorld. Please consider extending documentation.